de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/resources/AccessTestResourceTest.java

```diff
@@ -59,6 +59,6 @@ public void testServerConnection() {
 		String expectedJson = "{\"UUID\":\"data\"}";
 		
 		assertEquals("Server response is correct", expectedResponse, serverResponse);
-		assertEquals("Server json paylaod is correct", expectedJson, serverJson);
+		assertEquals("Server json paylaod is correct", expectedJson, serverJson.replaceAll("\\s+", ""));

```

Comment:
I now use the method from `de.dlr.sc.virsat.model.extension.tests.test.TestActivator`. But I am not sure if that is better

Reply:
I reverted the changes above because this class is deprecated and has to be deleted in https://github.com/virtualsatellite/VirtualSatellite4-Core/issues/597 anyways